protobuf-definitions/proto/mlagents_envs/communicator_objects/capabilities.proto

```diff
@@ -13,4 +13,7 @@ message UnityRLCapabilitiesProto {
 
     // concatenated PNG files for compressed visual observations with >3 channels.
     bool concatenatedPngObservations = 2;
+
+    // compression mapping for stacking compressed observations.
+    bool compressionMappings = 3;
```

Comment:
I guess we're not consistent about the name case ([protobuf style guide](https://developers.google.com/protocol-buffers/docs/style#message_and_field_names) says `snake_case` for field names). But we can at least be consistent for the variable names. Please `compressed_channel_mapping` or `compressedChannelMapping`.